packages/govuk-frontend/src/govuk/components/date-input/date-input.yaml

```
@@ -6,11 +6,11 @@ params:
   - name: namePrefix
     type: string
     required: false
-    description: Optional prefix. This is used to prefix each `item.name` using `-`.
+    description: Optional prefix. This is used to prefix each item `name` using `-`.
   - name: items
     type: array
     required: false
-    description: An array of input objects with name, value and classes.
+    description: The items of each date input component.
```

There is only 1 date input, so it doesn't make sense to say 'each date input component'. I'm also not sure 'items' is descriptive enough to help the user understand what this is.

Maybe:

```suggestion
    description: The inputs within the date input component.
```